Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -58,6 +55,9 @@ public protocol LoginToutViewModelOutputs {
   /// Emits an access token envelope that can be used to update the environment.
   var logIntoEnvironment: Signal<AccessTokenEnvelope, Never> { get }
 
+  /// Emits an image to be used as the navigation bar background.
+  var navigationBarBackgroundImage: Signal<UIImage?, Never> { get }
```

Comment:
Why is this needed? I noticed when testing that when scrolling, the navigation bar cuts off the scroll content view. Is that the desired behavior or do we want the navigation bar to fully transparent?

![Screen Shot 2020-03-30 at 3 03 49 PM](https://user-images.githubusercontent.com/3156796/77953545-7cf2b680-729b-11ea-8649-3155e96a4854.png)

Reply:
The `navigationBarBackgroundImage` to change color to `white` when going to signup and login screens (view with white background). I think that using your extension should fix this and this output shouldn't be necessary anymore.